test/sql/json/tpch_round_trip.test_slow

```diff
@@ -2,23 +2,24 @@
 # description: TPCH JSON round trip
 # group: [json]
 
-# TODO: implement COPY functions for JSON and do all tables more easily, we just do Q1 for now with from_json
-
 require tpch
 
 require json
 
+statement ok
+start transaction;
+
 statement ok
 call dbgen(sf=1)
 
 statement ok
-copy (select to_json(lineitem) from lineitem) to '__TEST_DIR__/lineitem.json' (format csv, quote '')
+copy lineitem to '__TEST_DIR__/lineitem.json'
```

Comment:
Can we read/write lineitem as an array-of-records as well, and verify that we are doing a streaming read by using `SET memory_limit=X`?

Reply:
I already tested reading/writing lineitem as ARRAY in `test_json_copy.test_slow`, added a memory limit there, works as expected.